packages/ui/src/2_molecules/Header/Header.module.css

```
@@ -1,5 +1,5 @@
 .header {
-  @apply bg-gray-90 h-16 w-full px-4 lg:px-11 flex items-center justify-between gap-4 relative;
+  @apply bg-gray-90 h-16 w-full px-4 lg:px-11 flex items-center justify-between gap-4 relative border border-gray-50;
```

```suggestion
  @apply bg-gray-90 h-16 w-full px-4 lg:px-11 flex items-center justify-between gap-4 relative border-b border-gray-50;
```
Border should be applied only to the bottom of the header.